added blackPanther OS support

blackPantherOS opened the pull request:
We added blackPanther support to your the beta, please merge if you agree on this patch

blackPantherOS committed:
added blackPanther OS support

Conan-Kudo commented:
@blackPantherOS Is there something wrong with the Mageia template for your use? The only difference is whether Python 2 is enabled by default or not...

hmikihth committed:
setup_method and test_rpm_name has been fixed

blackPantherOS committed:
Merge pull request #1 from hmikihth/master

setup_method and test_rpm_name has been fixed

hmikihth committed:
blackPanther OS spec file

hmikihth committed:
blackPanther OS spec file

blackPantherOS committed:
Merge pull request #2 from hmikihth/master

blackPanther OS spec file

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #3 from hmikihth/patch-1

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #4 from hmikihth/patch-2

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #5 from hmikihth/master

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge branch 'master' into master

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #6 from hmikihth/master

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #7 from hmikihth/master

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #8 from hmikihth/master

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge pull request #9 from hmikihth/patch-3

Update python-Jinja2_blackpantheros_py23.spec

hmikihth committed:
Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Merge branch 'master' into master

blackPantherOS committed:
Merge pull request #10 from hmikihth/master

Update python-Jinja2_blackpantheros_py23.spec

blackPantherOS committed:
Missing items fix

blackPantherOS committed:
Mistake fix

blackPantherOS committed:
Test package names add

blackPantherOS committed:
specfile clean

blackPantherOS committed:
specfile fix

blackPantherOS committed:
Upper name fix

blackPantherOS committed:
null

blackPantherOS committed:
Update name_convertor.py

blackPantherOS committed:
Update test_name_convertor.py

blackPantherOS committed:
Update README.md

blackPantherOS committed:
Merge pull request #11 from fedora-python/master

.

blackPantherOS committed:
Update tox.ini

blackPantherOS committed:
pyparsing

File "/tmp/easy_install-xTr1u0/pyparsing-3.0.0a1/pyparsing/core.py", line 166
    nonlocal found_arity, limit
                       ^
SyntaxError: invalid syntax
ERROR: InvocationError for command '/home/travis/build/fedora-python/pyp2rpm/.tox/py27/bin/python setup.py test --addopts -vv' (exited with code 1)

blackPantherOS committed:
Update tox.ini

ghost commented:
> @blackPantherOS Is there something wrong with the Mageia template for your use? The only difference is whether Python 2 is enabled by default or not...

The python2 test used wrong pyparsing. We had to add the pyparsing to the deps in the tox.ini file.

blackPantherOS committed:
Important XMLRPC time fix

blackPantherOS committed:
XSS Security hotfix  

Cross-Site Scripting (XSS) attacks. Refer: https://cwe.mitre.org/data/definitions/79.html

hmikihth committed:
Update convertor.py

Merged bP and Fedora modifications.

blackPantherOS committed:
Merge pull request #12 from hmikihth/master

Update convertor.py

blackPantherOS committed:
Merge branch 'fedora-python:main' into master

blackPantherOS committed:
Update version.py

3.3.11

blackPantherOS committed:
Update CHANGELOG

blackPantherOS committed:
Update CHANGELOG

blackPantherOS committed:
Updated to new version, and style fix

blackPantherOS committed:
Updated to new style
